repository_service_tuf_api/bootstrap.py

```diff
@@ -152,7 +152,13 @@ class Settings(BaseModel):
 
 class BootstrapPayload(BaseModel):
     settings: Settings
-    metadata: Dict[str, TUFMetadata]
+    metadata: Dict[
+        Literal[
+            Roles.ROOT.value,
+            Roles.TARGETS.value,
```

Comment:
I suggest we keep only `Roles.ROOT.value`

```suggestion
            Roles.ROOT.value,
```